controlpanel/api/models/dashboard.py

```diff
@@ -38,11 +40,29 @@ def arn(self):
     def is_admin(self, user):
         return self.admins.filter(pk=user.pk).exists()
 
-    def add_customers(self, emails):
+    def add_customers(self, emails, inviter_email):
+        notifications_client = NotificationsAPIClient(settings.NOTIFY_API_KEY)
+        not_notified = []
         for email in emails:
             viewer, _ = DashboardViewer.objects.get_or_create(email=email.lower())
             self.viewers.add(viewer)
 
+            try:
+                notifications_client.send_email_notification(
+                    email_address=email,
+                    template_id=settings.NOTIFY_TEMPLATE_ID,
+                    personalisation={
+                        "dashboard": self.name,
+                        "dashboard_link": self.url,
+                        "dashboard_home": settings.DASHBOARD_SERVICE_URL,
+                        "dashboard_admin": inviter_email,
+                    },
+                )
+            except HTTPError:
+                not_notified.append(email)
```

Comment:
If we get errors we want them to be raised in sentry, so we should use `sentry_sdk.capture_exception` here.

I'm also wondering about using Celery to send these notifications, as then we can retry sending a notification in the event of some failures. [Looking at the docs](https://docs.notifications.service.gov.uk/python.html#general-errors), it says that if you receive a 500 error you should retry, so we could check the code. So we could check the error code in an exception, and for a 500 error let it raise so that the task gets retried.

This might be something to look at adding later though, as there are downsides to this approach (not as easy to notify the admin something went wrong)